forward_engineering/ddlProvider/ddlProvider.js

```diff
@@ -1,13 +1,76 @@
 const templates = require('./ddlTemplates');
-const { getFullEntityName } = require('../utils/general');
+const {
+	getFullEntityName,
+	getName,
+	replaceSpaceWithUnderscore,
+	prepareName,
+	getContainerName,
+	getEntityData,
+	getEntityProperties,
+	wrapInBrackets,
+} = require('../utils/general');
+const { getViewTagsStatement } = require('../helpers/unityTagsHelper');
+const { getTablePropertiesClause } = require('../helpers/tableHelper');
+const viewHelper = require('../helpers/viewHelper');
 
 module.exports = app => {
 	const { assignTemplates } = app.require('@hackolade/ddl-fe-utils');
+	const _ = app.require('lodash');
```

Comment:
@Nightlngale ideally getting deps like this should be migrated to direct deps in the plugin when possible :)

Reply:
@bigorn0 I couldn’t agree more. But here I was trying to touch a small part of code related only to "Create view" script

Reply:
I agree with Ugo it's a good opportunity to fix it. But I would propose fixing it in a separate PR. @Nightlngale could you please create PR with this tech improvement?

Reply:
@bigorn0 @VitaliiBedletskyi yes, I will add this improvement in scope of the separate PR